src/components/pages/Offers/Offers.jsx

```diff
@@ -151,6 +151,10 @@ class Offers extends PureComponent {
     const { venueId, offererId } = apiParams
     const { nameSearchValue, page, pageCount, isLoading } = this.state
     const createOfferTo = createLinkToOfferCreation(venueId, offererId)
+    const backTo = {
+      path: [location.pathname, location.search].join(''),
+      label: 'Offers',
```

Comment:
```suggestion
      label: 'Offres',
```

Vu que c'est ce qui est affiché, ça doit être en français

Reply:
Oups, il faut que je m'habitue à avoir du francais dans le code.